jaeger/charts/jaeger/templates/rbac.yaml

```diff
@@ -37,6 +39,8 @@ kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: linkerd-jaeger-{{.Values.namespace}}-proxy-mutator
+  labels:
+    linkerd.io/extension: linkerd-jaeger
```

Comment:
I see that these are added for the cluster-wide resources. Should we also have these labels for deployments too? so that they are all easier to discover and perform these kinds of operations 🤔

Reply:
Yea I think it's something good to keep in mind; it definitely wouldn't hurt at least. If this is the only operation where we care about this for now though, then having the namespace to delete is enough.

Reply:
☝️